### DISCUSSION TITLE
Probes cannot be created with ssl skip

### DISCUSSION THREAD

**Discussion by amoughit** [litmuschaos/litmus#4477] [General]
When I create a HTTP probe, I do select the skipp verify ssl option but at the end it gets created with "N/A", I've also tried to modify but it didn't work. Is this a known issue ?  

![image](https://github.com/litmuschaos/litmus/assets/65241517/4c6c92a6-5f79-4806-a011-4e98ae250a43)

**Comment by prithvi1307** [comment]
@abdnah can you create this as an issue and also post this on the Litmus slack channel for quicker resolution? Although i am tagging @ispeakc0de for possible response here

**Comment by S-ayanide** [comment]
Hey @abdnah can you please verify which version of Litmus you are using? This bug was fixed in v3.4.0 and above. If your version is older, the first suggestion would be to upgrade to a version mentioned or higher. 

If you are already in v3.4.0 can you please create another one with the same metadata and share the payload from the network tab!

**Reply by amoughit** [reply]
Hi @prithvi1307, @S-ayanide, I'm using v3.3.0. I wasn't able to upgrade to v3.4.0 due to mongodb errors...

**Reply by amoughit** [reply]
https://github.com/litmuschaos/litmus/discussions/4476

**Reply by ispeakc0de** [reply]
Responded to 4476 issue
